### PR TITLE
Expense Splitter Website

### PR DESCRIPTION
**Pull request by vivekvardhan2810**
Closes:  #530

- **Title** : Expense Splitter Website
- **Name:** :  Vivek Vardhan
- **Idenitfy yourself:** :  GSSOC

### Describe the add-ons or changes you've made 📃

This advanced expense splitter website allows users to input the number of participants, dynamically add their names, and record expenses with details on who paid. It features an option to upload bill images and calculates how much each participant owes or is owed, ensuring all components are responsive and visually appealing.

### How Has This Been Tested? ⚙️

I have Tested This in my local machine

### Screenshots 📷

Screenshot

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] My changes generate no new warnings.

### CODE REVIEW

**Comment by vivekvardhan2810** [comment]
@Avdhesh-Varshney Please review and Merge the Pull Request

File: Vanilla-JS-Projects/Intermediate/Expense-Splitter-Website/Screenshot.webp
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Expense-Splitter-Website/Screenshot.webp:1]: It's `screenshot` not `Screenshot`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
resolve merge conflicts

**Comment by vivekvardhan2810** [comment]
@Avdhesh-Varshney I have made the changes requested

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**